PKGBUILD

```diff
@@ -1,25 +1,18 @@
 # Maintainer: Florian Moser <arch@famoser.ch>
 
 pkgname=symfony-cli
-pkgrel=1
+pkgrel=2
 pkgver=4.11.3
 pkgdesc="The Symfony client helps developers create and manage Symfony applications."
 url="https://symfony.com/"
 arch=('x86_64')
-license=('custom')
-install=''
+license=('MIT')
```

Comment:
Is the license here concerning the AUR package definition (which indeed is MIT) or concerning the package itself (which I do not know the license of)? The package itself is proprietary (https://symfony.com/legal/cli-eula).

Reply:
Good point! I just checked and it seems it's the licensing of the software itself, so yeah it should remain `custom`. Gonna change it.

However, from most of the proprietary software I've checked on the AUR (see how [spotify](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=spotify#n92) does it), it seems that your `LICENSE` file should be the license of the software, to then copy it in the `/usr/share/licenses` directory. Are you fine if I replace it?

Reply:
The spotify link is handy, thanks!

Not sure however for this package this the cleanest approach, as the AUR package definition by itself does not fall under the symfony CLI license.

I would instead create a new file for this (`Symfony CLI End-User License Agreement`) and copy this into the correct folder with `install -Dm 644 "Symfony CLI End-User License Agreement" "${pkgdir}/usr/share/licenses/${pkgname}/Symfony CLI End-User License Agreement"`. 

Additionally, I would place a notice in the `post_install()` that the user "accepts the terms and conditions of this Agreement by using the Application" or something along these lines. 

I can also do this by myself and would now merge your changes; your choice.

Reply:
Fixed! :slightly_smiling_face: